plugins/webapi/api.yaml

```diff
@@ -0,0 +1,260 @@
+openapi: 3.0.0
+info:
+  title: GoShimmer API
+  description: Optional multiline or single-line description in [CommonMark](http://commonmark.org/help/) or HTML.
+  version: 0.1.0
+servers:
+  - url: http://localhost:8080
+tags:
+  - name: "transactions"
+  - name: "neighbors"
+  - name: "misc"
+
+paths:
+  /broadcastData:
+    post:
+      tags:
+        - transactions
+      summary: creates a data (0-value) transaction
+      description: broadcastData creates a data (0-value) transaction given an input of bytes and broadcasts it to the node's neighbors. It returns the transaction hash if successful.
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                address:
+                  type: string
+                data:
+                  type: string
+      responses:
+        200:
+          description: transaction successfully created
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  hash:
+                    type: string
+        400:
+          description: invalid address, data too long or invalid trinary
+  /findTransactions:
+    post:
+      tags:
+        - transactions
+      summary: returns array of transaction hashes for given addresses
+      description: returns the array of transaction hashes for the given addresses (in the same order as the parameters). If a node doesn't have any transaction hash for a given address in its ledger, the value at the index of that address is empty.
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                addresses:
+                  type: array
+                  items:
+                    type: string
+      responses:
+        '200':
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  transactions:
+                    type: array
+                    items:
+                      type: array
+                      items:
+                        type: string
+  /getNeighbors:
+    get:
+      tags:
+        - neighbors
+      summary: returns the chosen and accepted neighbors of the node
+      description: Returns the chosen and accepted neighbors of the node. Optionally, a parameter `known=1` can be provided so that all known hosts are returned as well.
```

Comment:
```suggestion
      description: Returns the chosen and accepted neighbors of the node. Optionally, a parameter `known=1` can be provided so that all known peers are returned as well.
```